addons/pos_self_order_adyen/controllers/main.py

```diff
@@ -33,6 +33,7 @@ def notification(self):
                     'pos_order_id': order_sudo.id
                 })
                 order_sudo.action_pos_order_paid()
+                request.env['pos_preparation_display.order'].sudo().process_order(order_sudo.id)
```

Comment:
Again, unfortunately `pos_self_order_adyen` do not depend on `pos_preparation_display`